packages/build-tooling/package.json

```diff
@@ -68,7 +69,7 @@
     "ansis": "catalog:*",
     "chokidar": "catalog:*",
     "esbuild": "catalog:*",
-    "glob": "^10.3.10",
+    "fast-glob": "^3.3.2",
```

Comment:
I have seen that `fast-glob` is used in other packages so I replace glob with it.